.github/workflows/release.yml

```diff
@@ -122,4 +95,28 @@ jobs:
           ${{steps.get-checksums-from-draft-release.outputs.result}}
           EOF
           )
-          
\ No newline at end of file
+          
+      - name: Run Test cases
+        run: |
+          # Setup minikube
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+          install minikube-linux-amd64 /usr/local/bin/minikube
+          minikube start --driver=docker --memory 4096
+          eval $(minikube docker-env --shell=bash)
+
+          # Ensure that there is no existing kapp installed
+          rm -f /tmp/bin/kapp
+
+          # Build kapp binary
+          set -e -x
+          VERSION=`echo ${{ github.ref }}  | grep -Eo '[0-9].*'`
+          ./hack/build.sh
+
+          # Add binary to the path
+          mkdir bin
+          mv kapp bin
+          PATH=$PATH:$PWD/bin
+          echo $PATH
+
+          # Run test cases
+          ./hack/test-external.sh
```

Comment:
Add a new line at the end of file?

Reply:
Not necessarily. Deliberately removed it as it was difficult to remember it every time so that diff didn't come.

Reply:
@rohitagg2020 I would insist on this, mainly because it sets GitHub off and something we stick to.
the formatter would be doing this for all the `.go` files already.

Reply:
There *is* a line at the end of the file. Am I missing something here?

Reply:
I have checked just now. @100mik There were two new lines at the end. I have removed one extra line at the end. We already have files ending like these in the other [workflows](https://github.com/carvel-dev/kapp/tree/develop/.github/workflows)

Reply:
Ah alright, I think the file didn't have one before, GitHub threw me off with a warning.